### PR TITLE
SI-2710 Remove experimental VSS Elastic handlers

### PR DESCRIPTION
**Pull request by elffjs**
# Proposed Changes

### CODE REVIEW

**Comment by Allyson-English** [comment]
lgtm @adityap92 tagging you for visibility-- this is deprecating the `/v2/vehicle/:tokenID/history` endpoint. seems like we're grabbing privileged history [here](https://github.com/DIMO-Network/device-data-api/blob/main/cmd/device-data-api/api.go#L107) on the `/v1` endpoint so I don't think mobile will be impacted

**Review by Allyson-English — APPROVED** [review]

**Comment by elffjs** [comment]
The v2 index never had data on prod, yeah/

File: internal/services/elastic/elastic.go [hunk 1]
```diff
@@ -197,158 +193,3 @@ func (s *Service) GetTotalDistanceDriven(ctx context.Context, deviceID string) (
 
 	return body, nil
 }
-
```
Comment (KevinJoiner) [inline internal/services/elastic/elastic.go:200]: In this file, you can delete these as well
```go
type GetHistoryParams struct {...
	
func (g *GetHistoryParams) SetDefaultHistoryParams() {
```

**Review by KevinJoiner — APPROVED** [review]

File: internal/services/elastic/elastic.go (hunk 1 above)
Comment (elffjs) [reply internal/services/elastic/elastic.go:200]: Ah, thanks, careless of me.

**Pull request merged**